README.md

```diff
@@ -36,7 +36,13 @@ RoboHome is a SaaS tool that also integrates with Amazon's Echo to enable contro
 
 Docker support is currently on hold while the port to Laravel takes place.
```

Comment:
You can remove this line thanks to your work!

"Docker support is currently on hold while the port to Laravel takes place."